app/delivery/rover_routes.js

```diff
@@ -0,0 +1,20 @@
+import express from "express";
+import getMarsPhotos from "../useCases/fetch_rover_photos.js";
+import Exception from "../utils/exception.js";
+
+
+const roverRouter = express.Router();
+
+roverRouter.post('/rover', async (req, res, next) => {
+    try {
```

Comment:
Some places lack input validation and error handling at the API level. you can use validation libraries in such places to validate API input parameters before executing a request

Reply:
The addition of validation is planned for the next task